### PR TITLE
Signature

### PR DESCRIPTION
**Pull request by ctlong**

### CODE REVIEW

File: app/assets/javascripts/angular/controller/forms/updateForms.js.erb [hunk 1]
```diff
@@ -165,7 +163,14 @@ app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$ro
     });    
   }
 
-  //Drag and drop
+  //Set signature to true or false
+  $scope.signatureStatus = function() {
+    $http.patch(url + '/forms/' + $routeParams.id,$scope.form).success(function(response) {
+      console.log(response);
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/updateForms.js.erb:169]: only console.log?

File: app/assets/javascripts/angular/controller/forms/updateForms.js.erb [hunk 2]
```diff
@@ -165,7 +163,14 @@ app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$ro
     });    
   }
 
-  //Drag and drop
+  //Set signature to true or false
+  $scope.signatureStatus = function() {
+    $http.patch(url + '/forms/' + $routeParams.id,$scope.form).success(function(response) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/updateForms.js.erb:168]: what data are you modifying?

File: db/migrate/20150908071237_create_forms.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@ class CreateForms < ActiveRecord::Migration
   def change
     create_table :forms do |t|
       t.string :name
+      t.boolean :signature
```
Comment (h8rry) [inline db/migrate/20150908071237_create_forms.rb]: better naming would be preferred

**Comment by h8rry** [comment]
@ctlong please review

File: app/assets/javascripts/angular/controller/forms/updateForms.js.erb (hunk 2 above)
Comment (ctlong) [inline app/assets/javascripts/angular/controller/forms/updateForms.js.erb:168]: All I'm doing is updating the database to match the state of the checkbox, thats why there is nothing inside.

File: db/migrate/20150908071237_create_forms.rb (hunk 3 above)
Comment (ctlong) [inline db/migrate/20150908071237_create_forms.rb]: What's wrong with signature? Also already added that name everywhere.

Comment (h8rry) [inline db/migrate/20150908071237_create_forms.rb]: `signature_required` would be a more suitable name

**Comment by ctlong** [comment]
@ggharry Reviewed and edited.

**Comment by h8rry** [comment]
@ctlong can merge

**Pull request merged**